lib/render.js

```diff
@@ -66,6 +77,22 @@ function transfer(from, to, fromProp, toProp) {
 }
 
 /**
+ * Look through env variables array and return
+ * the variables from the `process.env` bject
+ *
+ * @return {Object}
+ */
+function resolveEnvVars() {
+  var obj = {};
+
+  _.map(module.exports.envVars, function (val) {
```

Comment:
`map` is for transforming an array, but this just side-effects. You could use `Array.prototype.forEach`, `_.each` or some other `_` method to better-communicate intent